core/templates/services/rte-helper-modal.controller.spec.ts

```diff
@@ -108,9 +108,10 @@ describe('RteHelperModalComponent', () => {
     beforeEach(() => {
       fixture = TestBed.createComponent(RteHelperModalComponent);
       component = fixture.componentInstance;
-      component.attrsCustomizationArgsDict = {
-        heading: 'This value is not default.',
-      };
+      (component.componentId = 'video'),
```

Comment:
Why not do these as two separate statements (separated by semicolon)?

Reply:
In fact I think it would be more readable if separated. I will change that!